src/app/service/scoap-squad/ScoapPoll.ts

```diff
@@ -0,0 +1,282 @@
+import { TextChannel, MessageReaction } from 'discord.js';
+import cloneDeep from 'lodash.clonedeep';
+import isEqual from 'lodash.isequal';
```

Comment:
Does this need to be imported in the `package.json`?

Reply:
i think so, added it. Although, could it be that this is installed with clonedeep as dependency?  I don't know how to tell. It appears in my yarn.lock file along with some other lodash modules which are also not in package.json. Also removed fastify/helmet/cors for now since we're not using it yet